### ISSUE TITLE
http destination does not compile on git head on SLES 11

### ISSUE THREAD

**Issue by czanik** [balabit/syslog-ng#1270]
## Platform

SLES 11 SP4

# Issue

When creating a dist tgz from git head and compiling it on SLES 11, it fails at compiling the http (curl) destination. The 3.8.1 release compiled fine in the same environment.

## Failure

```
[   72s]   CCLD     modules/afamqp/libafamqp.la
[   72s]   CC       modules/http/modules_http_libcurl_la-http.lo
[   72s] modules/http/http.c: In function 'http_dd_set_headers':
[   72s] modules/http/http.c:240: warning: implicit declaration of function 'g_list_free_full'
[   72s]   CC       modules/http/modules_http_libcurl_la-http-grammar.lo
[   73s] modules/http/http-grammar.y:34:26: error: http-grammar.h: No such file or directory
[   73s] make[2]: *** [modules/http/modules_http_libcurl_la-http-grammar.lo] Error 1
[   73s] make[1]: *** [all-recursive] Error 1
[   73s] make: *** [all] Error 2
```

**Comment by bazsi** [comment]
newer glib?


-- 
Bazsi

**Comment by gaborznagy** [comment]
Fixed by #1897
